frameworks/elastic/tests/test_tls.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import shakedown
+
+import sdk_cmd
+import sdk_install
+import sdk_plan
+import sdk_security
+from tests import config
+from tests.config import (
+    PACKAGE_NAME,
+    NO_INGEST_TASK_COUNT,
+)
+
+
+@pytest.fixture(scope='module', autouse=True)
+def client_over_tls():
+    """
+    This fixture forces all requests from config.py module to be issued over
+    TLS connection.
+    """
+    original_value = config.ELASTIC_TLS_ENABLED
```

Comment:
I'm not a big fan of global state, but I can see how this is the simplest way to get this done. What are our options otherwise?

This could cause some headaches if we, for example, decide to run tests in parallel.

Reply:
Looking at the changes, adding an argument to the CRUD operations should be sufficient and remove the need for the global variable.

Reply:
Changed.